service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/PlacementSubmittedEventControllerTest.java

```diff
@@ -253,8 +254,9 @@ void shouldSendEmailNotificationsWhenNewNoticeOfPlacementOrder() throws Notifica
                 dataCaptor.capture(),
                 eq(CASE_ID));
 
-            MapDifference<String, Object> difference = verifyNotificationSentToRepresentativesWhenOrderIssued(
-                dataCaptor, IssuedOrderType.NOTICE_OF_PLACEMENT_ORDER);
+            MapDifference<String, Object> difference = verifyNotification(dataCaptor,
+                () -> getExpectedParametersForRepresentatives(IssuedOrderType.NOTICE_OF_PLACEMENT_ORDER),
```

Comment:
Static import?

Reply:
Unfortunately already exists a static import of NOTICE_OF_PLACEMENT_ORDER from PlacementOrderAndNoticesType